demisto_sdk/commands/init/contribution_converter.py

```diff
@@ -379,7 +380,8 @@ def generate_readmes_for_new_content_pack(self, is_contribution=False):
                             self.generate_readme_for_pack_content_item(
                                 unified_file, is_contribution
                             )
-                            os.remove(unified_file)
+                            unified_file_path = Path(unified_file)
+                            Path.unlink(unified_file_path)
```

Comment:
```suggestion
                            Path(unified_file).unlink()
```
unlink doesn't take arguments